src/utilities/getMaskedAsset.ts

```diff
@@ -0,0 +1,17 @@
+import { config } from './config';
+
+/**
+ * @param mediaSrc - The Storyblok URL of the asset
+ * @returns The processed Storyblok asset URL that is masked by our asset CDN
+ */
+
+export const getMaskedAsset = (mediaSrc: string): string => {
+  if (!mediaSrc) {
+    return undefined;
+  }
+
+  const { imageService } = config;
+  const maskedUrl = mediaSrc.replace(imageService, `${config.assetCdn}a-us/`);
+
+  return maskedUrl;
+};
```

Comment:
Extract this function from getProcessedImage, so it can be applied to non-image assets to get the masked URL with our assets.stanford.edu CDN